docs/pages/access-controls/access-request-plugins/access-monitoring-rules.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Routing Access Requests to Access Plugins
+description: How to set up Teleport's Access Monitoring Rules
+---
+
+With Teleport's Access Monitoring Rules access request notifications
+can be routed to plugins based on based on several conditions.
+
+Currently only the Slack access plugin is supported.
+
+## Prerequisites
+
+- A Teleport Enterprise Cloud account.
```

Comment:
```suggestion
- A managed Teleport Enterprise account.
```
We're using "Teleport Enterprise" for both cloud and self-hosted accounts now, specifying whether it is managed outside the name of the product.